src/main/java/uk/gov/hmcts/reform/managecase/service/noc/ApprovalStatus.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.managecase.service.noc;
```

Comment:
Should this be in one of the model packages: e.g. `uk.gov.hmcts.reform.managecase.domain` alongside org policy etc..

Reply:
Agreed, will move

Reply:
👍  implemented